packages/mollie-integration/src/mollie-payment-strategy.ts

```diff
@@ -300,18 +283,18 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         );
     }
 
-    private _mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
+    private mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
```

Comment:
May I ask you to rename all function names to the same style (without `_` before the name)?

For example< you have `mountCardVerificationfields` and `_isHostedPaymentFormEnabled`.

Reply:
done